### PR TITLE
Basic HTTP forwarding to single URI

### PR DESCRIPTION
**Pull request by mwiniarski**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+
+[*.md]
+trim_trailing_whitespace = false
+
+[*.yml]
+indent_size = 2
```
Comment (buoto) [inline .editorconfig]: Pls add newline

**Comment by buoto** [comment]
Please fix scripts file permissions:
```
git checkout HEAD~ -- scripts/* 
```

**Comment by buoto** [comment]
Please fix scripts file permissions:
```
git checkout HEAD~ -- scripts/* 
```

File: Cargo.toml [hunk 2]
```diff
@@ -1,9 +1,11 @@
 [package]
 name = "balanser"
 version = "0.1.0"
-authors = ["Michał Błotniak <miblotniak@gmail.com>"]
+authors = ["Michał Błotniak <miblotniak@gmail.com>", "Michał Winiarski <michal.winiarsk@gmail.com>"]
```
Comment (Diewa) [inline Cargo.toml:4]: 😢

Comment (buoto) [reply Cargo.toml:4]: Please add yourself in your review :)

**Review by buoto — APPROVED** [review]

**Review by mafiesto4 — APPROVED** [review]

**Pull request merged**